### ISSUE TITLE
Web App blank display

### ISSUE THREAD

**Issue by bikeoid** [codeforgreenville/trolley-tracker#120]
Lori (the trolley driver) has tried to use the web-based tracker from both TrackTheTrolley.com and directly from http://linktheorigi.webfactional.com/trolley/ .   The result is that the map display area is empty (blank, no route, trolley, or map) 2 out of the 3 times she tries it.   Last year, her iPhone screen was severely cracked, and I chalked it up to perhaps the broken screen appearing as a continuous left or right mouse click.   Chrome gave the same blank result.  But she replaced the iPhone over New Years, and the result is the same on her new phone.   Verizon is her carrier.

She is happy the the iPhone app is now available, and that solves her problem.   She will be one to use the app to pace her trolley 180 degrees out of phase from the other trolley on the route!

I don't expect this will be a big problem since most people will use the Apps, but just a note in case it happens to others.   It has never happened on my iPhone with Safari.

**Comment by ajhodges** [comment]
@dacohenii @Linktheoriginal any ideas?

**Comment by Linktheoriginal** [comment]
It makes sense that a continuous click would be telling the map to move, which would load blank white until the picture loaded.  As for the new phone, I'm not sure - were you able to look at what she was doing?  I haven't set up any logging/diagnostic tools on the webserver, as I haven't heard any other reports of issues like that.  It could be an issue with Mapbox limiting something (since they host the map tiles), but I don't think that would be an intermittent issue.

**Comment by dacohenii** [comment]
I've also never had this issue on Safari for iPhone, but I agree that this should be looked into.

You mentioned that it worked 2 of 3 times. Were all 3 tries at different times, or at the same time?

It may be useful to know what version of iOS she has and whether or not she's using a content blocker. 

However, I'm really not sure what would be causing it to only work intermittently. Let me know if you find out any more information that could help me replicate the issue. In the meantime, I'll be on the lookout.

**Comment by bikeoid** [comment]
I agree that it seems like a content blocker or spyware or AntiVirus proxy.   Does Safari have a developers console to indicate something about the page?

**Comment by Linktheoriginal** [comment]
All the debugging I've done on safari web stuff I've needed to inject firebug lite into the page via JavaScript.  Most other debugging you have to plug it in to a Mac/Desktop to work on.
